src/components/Map/StandardMap.vue

```diff
@@ -86,8 +87,25 @@ export default {
       if (this.selectedMarkers.length > 0) {
         return L.latLngBounds(this.selectedMarkers.map(m => m.latLng)).pad(0.2)
       }
-      else {
-        return L.latLngBounds([{ lat: '49.8990022441358', lng: '8.66415739059448' }])
+    },
+    center () {
+      if (!this.bounds) {
+        if (this.defaultCenter) {
+          return [this.defaultCenter.latitude, this.defaultCenter.longitude]
+        }
+        else {
+          return ['49.8990022441358', '8.66415739059448']
```

Comment:
Should be most amusing place possible...

Reply:
it's still Darmstadt 🍂